### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bish0polis**
- 3: add words to eliminate comma splice
 - 5: add word to fix symmetry and remove ambiguity
 - 13: comma splice
 - 14: comma splice
 - 15: 'set up' and 'setup' are not the same
 - 19: comma splice and clarification of email address usage
 - 21: comma splice
 - 25: colon to delineate supporting independent clause
 - 33: colon to delineate supporting independent clause
 - 38: comma splice
 - 39: colon to delineate supporting independent clause; also 'set up' and 'setup' are not the same, just as 'back up' and 'backup' are not the same

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,42 @@
 # Haven
 
-[Haven](https://havenweb.org) is a simple blog application (formerly called SimpleBlog) built with Ruby on Rails, some motivating philosophies:
+[Haven](https://havenweb.org) is a simple blog application (formerly called SimpleBlog) built with Ruby on Rails, based on some motivating philosophies:
```
Comment (mawise) [inline README.md:3]: How about "here are" instead of "based on"

**Review by mawise — CHANGES_REQUESTED** [review]
Thanks for the edits!

File: README.md (hunk 1 above)
Comment (bish0polis) [reply README.md:3]: It would still be a comma splice, then.

Comment (mawise) [reply README.md:3]: We can split it with a period, make "Here are" the start of a new sentence.  Does that work?  Really appreciate your help cleaning up the readme!

**Comment by mawise** [comment]
Thank you!

**Pull request closed without merging**